app/containers/Tracks/index.js

```diff
@@ -152,14 +162,10 @@ export function Track({dispatchTrackDetails,
 
   return (
     <Container>
-
-       <RightContent>
-        <StyledT onClick={handleStoriesClick} data-testid="redirect" id="stories" />
-      </RightContent>
       <CustomCard maxwidth={maxwidth}>
-        <CustomCardHeader title={translate('track_search')} />
+        <CustomCardHeader title={translate('Track Search')} />
```

Comment:
These ids should be the en json field ids. Therefore, follow the convention of having an underscore text like search_track_details and track_search